groups/validate.go

```diff
@@ -73,27 +91,39 @@ func TestMain(m *testing.M) {
 		}
 	}
 
-	if err := cfg.Load(*groupsPath, &rConfig); err != nil {
+	if err := groupsConfig.Load(*groupsPath, &restrictionsConfig); err != nil {
 		fmt.Printf("Could not load groups config: %v\n", err)
 		os.Exit(1)
 	}
-	os.Exit(m.Run())
+
+	// Run all the Verification Logic
+	VerifyMergedGroupsConfig(config)
+	VerifyStagingEmailLength(config)
+	VerifyDescriptionLength(config)
+	VerifyGroupConventions(config)
+	VerifyK8sInfraGroupConventions(config)
+	VerifyK8sInfraRBACGroupConventions(config)
+	VerifySecurityResponseCommitteeGroups(config)
```

Comment:
It feels like it might make sense to use the `config.SkipKubernetesIOTests` bool here to make it clear which verifications are skipped (rather than doing it inside the methods).

Reply:
Fixed